### ISSUE TITLE
Type 'typeof import("...@octokit/rest/index") is not a constructor function type

### ISSUE THREAD

**Issue by nwtgck** [nwtgck/actions-netlify#2]
This problem is only related to users deploying this repo.

## Develop problem

Using `npm run build`, you will have the following error.

```
[actions-netlify]$ npm run build

> actions-netlify@0.1.2-SNAPSHOT build /...
> tsc

node_modules/@actions/github/lib/github.d.ts:5:37 - error TS2507: Type 'typeof import("/.../node_modules/@octokit/rest/index")' is not a constructor function type.

5 export declare class GitHub extends Octokit {
                                      ~~~~~~~

node_modules/@actions/github/lib/github.d.ts:15:52 - error TS2694: Namespace '"/.../node_modules/@octokit/rest/index"' has no exported member 'Options'.

15     constructor(token: string, opts?: Omit<Octokit.Options, 'auth'>);
                                                      ~~~~~~~

node_modules/@actions/github/lib/github.d.ts:16:31 - error TS2694: Namespace '"/.../node_modules/@octokit/rest/index"' has no exported member 'Options'.

16     constructor(opts: Octokit.Options);
                                 ~~~~~~~

src/main.ts:33:26 - error TS2339: Property 'issues' does not exist on type 'GitHub'.

33       await githubClient.issues.createComment({
                            ~~~~~~


Found 4 errors.
```

## Workaround

You can directly modify `./node_modules/@actions/github/lib/github.d.ts` as follows.

```diff
- import Octokit from '@octokit/rest';
+ import {Octokit} from '@octokit/rest';
```

**Comment by nwtgck** [comment]
This should be resolved by https://github.com/actions/toolkit/pull/332.

**Comment by nwtgck** [comment]
Solved by 245a3b6307f78056481d99ccd1a6bf8d84fa7597. The solution was from https://github.com/actions/toolkit/pull/332#issuecomment-582273380. Thanks.
